lib/arm/crc32_impl.h

```diff
@@ -3,6 +3,7 @@
  *
  * Copyright 2017 Jun He <jun.he@linaro.org>
  * Copyright 2018 Eric Biggers
+ * Copyright 2019 Greg V
```

Comment:
Can you please include your full last name?

Reply:
No. Actually, I don't really need a copyright line..